Cabozantinib.json

```diff
@@ -66216,1906 +66216,6 @@
         }
       ]
     },
-    {
```

Comment:
3 Parameter identifications (PIs) were corrupted due to https://github.com/Open-Systems-Pharmacology/PK-Sim/issues/1767
All 3 PIs did not reference any simulation (I guess PIs were created once and then the simulations were removed from the project) so I think there is no loss of information if they are removed.
There are more PIs of this kind (no referenced simulations) but only those 3 caused an error when loading from snapshot